intra_process_demo/src/image_pipeline/camera_node.cpp

```diff
@@ -12,12 +12,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <memory>
+
 #include "image_pipeline/camera_node.hpp"
 
 #include <rclcpp/rclcpp.hpp>
```

Comment:
Should we change to double-quotes here?

Reply:
It's not necessary to satisfy cpplint. Basically, if the header has the suffix ".h", then we need to use double-quotes (or move it to the top of the file) since cpplint assumes it is a C system header.

It some places where there was a mix of headers with the suffix ".h" and ".hpp", I used double-quotes everywhere for aesthetics :shrug:

Reply:
All right.  I actually don't have a strong position here, I was just going for consistency.  I'll approve as-is.

Reply:
Switched to double-quotes for backwards-compatibility with the linter (1349899).